test/test_integration_cluster.rb

```diff
@@ -2,102 +2,122 @@
 require_relative "helpers/integration"
 
 class TestIntegrationCluster < TestIntegration
-  parallelize_me!
-
-  DARWIN = !!RUBY_PLATFORM[/darwin/]
-
-  def setup
-    skip NO_FORK_MSG unless HAS_FORK
-    super
-  end
+  parallelize_me! unless Puma.jruby?
```

Comment:
This line isn't ever run on JRuby anyway, is it?

Reply:
Ok, so maybe I forgot that JRuby doesn't support fork.  I did that before I decided I was tired of seeing all the skip messages and added the class end statement conditional.  I'll remove.